### PR TITLE
styling: updated the navigation and form area

### PR DESCRIPTION
**Pull request by impelcrypto**
* updated the navigation and form area
  * ref: https://github.com/AstarNetwork/crowdloan-app/pull/73#issuecomment-1108200618

=Before=

image

image

=After= 

image

image

### CODE REVIEW

File: src/components/shiden/StakeForm.vue [hunk 1]
```diff
@@ -72,17 +72,46 @@
...
+        </div>
+        <div class="box--disclaimer-line">
+          <div class="row--disclaimer">
+            <div>
+              <span class="text--form-order">1.</span>
+            </div>
+            <div>
+              <div>
+                <span class="text--form"
+                  >KSM will be locked for 48weeks. Rewards SDN has
```
Comment (hoonsubin) [inline src/components/shiden/StakeForm.vue:87]: need space between 48 and weeks.

File: src/components/shiden/StakeForm.vue [hunk 2]
```diff
@@ -72,17 +72,46 @@
...
+                target="_blank"
+                rel="noopener noreferrer"
+                >Learn more.</a
+              >
+            </div>
+          </div>
+          <div class="row--disclaimer">
+            <span class="text--form-order">2.</span>
+            <span class="text--form"
+              >Please DO NOT join from Ledger/Lockdrop accounts. We might not
```
Comment (hoonsubin) [inline src/components/shiden/StakeForm.vue:103]: should be 
> Please DO NOT join from a Ledger/Lockdrop account. We cannot provide any support for those accounts.

**Review by hoonsubin — CHANGES_REQUESTED** [review]

File: src/components/shiden/StakeForm.vue (hunk 2 above)
Comment (impelcrypto) [reply src/components/shiden/StakeForm.vue]: Fixed. Thank you!

image

File: src/components/shiden/StakeForm.vue (hunk 1 above)
Comment (impelcrypto) [reply src/components/shiden/StakeForm.vue]: Fixed.

**Review by hoonsubin — APPROVED** [review]
LGTM

**Comment by Kahonnohak** [comment]
@hoonsubin 
The other text area hasn't been reviewed yet and the header link of FAQ is still with the previous CL. Who would be the person to give the contents writing please?

@impelcrypto Thanks!

**Review by Kahonnohak — APPROVED** [review]

**Pull request merged**